File: pbinfo_probs/varste_copii.cpp
#include <iostream>

using namespace std;
int a, b, x;
int main()
{
    cin>>a>>b;
    if(a>b)
        {
            x=a-b;
            cout<<"Primul copil este mai mare cu "<<x<<" ani";
        }
    else if(b>a)
        {
            x=b-a;
            cout<<"Al doilea copil este mai mare cu "<<x<<" ani";
        }
    else
        cout<<"Copiii au varste egale";
    return 0;
}

File: pbinfo/varste_copii.py
x=raw_input()
x=x.split()
a=int(x[0])
b=int(x[1])

if a==b:
    print "Copiii au varste egale"
elif a>b:
    print "Primul copil este mai mare cu " + str(a-b) + " ani"
else:
    print "Al doilea copil este mai mare cu " + str(b-a) + " ani"
